### PR TITLE
Add docker image to management API

### PR DESCRIPTION
**Pull request by marrobi**
# PR for issue 73

## What is being addressed

Added docker image to management API and exposed through app gateway

## How is this addressed

- Added docker settings to Azure Web App
- Added path mapping for /api to app gateway to web app back end
- upped TF provider version to more recent version

image

### CODE REVIEW

File: templates/core/terraform/appgateway/appgateway.tf [hunk 1]
```diff
@@ -14,7 +14,7 @@ resource "azurerm_application_gateway" "agw" {
   sku {
     name     = "Standard_v2"
     tier     = "Standard_v2"
-    capacity = 2
+    capacity = 1
```
Comment (TessFerrandez) [inline templates/core/terraform/appgateway/appgateway.tf:17]: Is this for the docker container? or if not, can you mention why this change is made?

**Review by TessFerrandez — COMMENTED** [review]
We should also update the dev setup documentation on how this is used

**Review by TessFerrandez — APPROVED** [review]

File: templates/core/terraform/appgateway/appgateway.tf (hunk 1 above)
Comment (marrobi) [reply templates/core/terraform/appgateway/appgateway.tf:17]: Ah, not really part of this PR, we only need 1 - half the cost - can put it back or in another PR if think I should.

Comment (TessFerrandez) [reply templates/core/terraform/appgateway/appgateway.tf:17]: No - it's cool, was just curious :)

**Review by deniscep — APPROVED** [review]
